pbft/database_pbft_service.cpp

```diff
@@ -143,6 +145,9 @@ database_pbft_service::process_awaiting_operations()
                 this->crud->handle_request(op_it->second->get_request().sender(), request, nullptr);
             }
 
+            // update stats...
+            this->monitor->finish_timer(bzn::statistic::request_latency, op_it->second->get_request_hash());
```

Comment:
Cool, latency recorded here after we finish handling the request. But, should it happen before the request is handled? Right after we get the entire response? That is, do we want to include the processing of the request in the time? (This is just asking for my own curiosity. )